server/workflowarchive/archived_workflow_server.go

```diff
@@ -47,9 +48,32 @@ func (w *archivedWorkflowServer) ListArchivedWorkflows(ctx context.Context, req
 	if offset < 0 {
 		return nil, status.Error(codes.InvalidArgument, "listOptions.continue must >= 0")
 	}
+
 	namespace := ""
-	if strings.HasPrefix(options.FieldSelector, "metadata.namespace=") {
-		namespace = strings.TrimPrefix(options.FieldSelector, "metadata.namespace=")
+	minStartedAt := time.Time{}
+	maxStartedAt := time.Time{}
+	selectors := strings.Split(options.FieldSelector, ",")
+	for _, selector := range selectors {
```

Comment:
Can not use k8s `fields.ParseSelector()` because it only supports 3 operators: `=`, `==` and `!=`.